suites/apis/rasAuthN.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable codeceptjs/no-skipped-tests */
```

Comment:
Do we want to have these new scenarios in a whole new/separate file?
`rasAuthN.js` or should we just append these scenarios to the existing `rasIntegrationTest.js` script?

Reply:
These are idempotent tests ... we can merge back later by refactoring the existing tests to run independently in the same way. For now, I will leave it here.